typescript-conditionals/main.ts

```diff
@@ -0,0 +1,140 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(x: number): boolean {
+  if (x < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const isUnderFiveResult: boolean = isUnderFive(3);
+console.log('value of isUnderFiveResult:', isUnderFiveResult);
+
+function isEvenNumber(x: number): boolean {
+  if (x % 2) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const isEvenNumberResult: boolean = isEvenNumber(10);
+console.log('value of isEvenNumber:', isEvenNumberResult);
```

Comment:
This condition is incorrect. If you look at your `console.log` for `value of isEvenNumber` you'll see that when you passed in 10, it says `false`, but 10 is an even number.

If you `console.log(x % 2)` in the function, what do you get? Is that what you should get with an even number? 🤔